<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cdegs_main</class>
 <widget class="QMainWindow" name="cdegs_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>603</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Narrow</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CDEGS Aid</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>2</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>2</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>2</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>2</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QTabWidget" name="tabProjects">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionSave_Project_as"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionNew_Config"/>
    <addaction name="actionOpen_Config"/>
    <addaction name="actionSave_Config"/>
    <addaction name="actionClose_Config"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExport_Config"/>
    <addaction name="actionExport_Config_as"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
   <addaction name="menuExport"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About..</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project..</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSave_Project_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project as..</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Project</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew_Config">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>New Config</string>
   </property>
  </action>
  <action name="actionOpen_Config">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Config..</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionClose_Config">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Config</string>
   </property>
  </action>
  <action name="actionExport_Config">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Config</string>
   </property>
  </action>
  <action name="actionExport_Config_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Config as..</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>cdegs_main</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
